packages/backend-test-utils/src/database/types.ts

```diff
@@ -45,6 +45,8 @@ export type Instance = {
   stopContainer?: () => Promise<void>;
   databaseManager: DatabaseManager;
   connections: Array<Knex>;
+  dropDatabasesInShutdown: boolean;
```

Comment:
```suggestion
  dropDatabase?: () => Promise<void>;
```

seems like the better contract, then each database driver can both choose whether to return it, and how it's implemented.

Then the code above would be exactly like the stopContainer code

```ts
      try {
        await dropDatabase?.();
      } catch (error) {
        console.warn(`TestDatabases: Failed to drop database`, {
          databaseManager,
          error,
        });
      }
```

Reply:
Modified the code to use callback-style contract.